src/components/ButtonPanel.js

```diff
@@ -5,7 +5,7 @@ import Button from './Button';
 
 export default class ButtonPanel extends React.Component {
   renderButton(name) {
-    const wide = name === '0' ? '50%' : '25%';
+    const wide = name === '0' ? true : false;
```

Comment:
If the wide is boolean you could just do `wide ?  '50%' : '25%' `

Reply:
Thanks for pointing it out, it is resolved!